src/openloco/openloco.vcxproj

```diff
@@ -15,7 +15,110 @@
     </ProjectConfiguration>
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="**\*.cpp" Exclude="version.cpp" />
+    <ClCompile Include="audio\audio.cpp" />
```

Comment:
Please revert the (accidental?) changes to this file. We use wildcards on purpose.

Reply:
Sorry I hadn't noticed that I'd commited the changes to this